test/blackbox-tests/test-cases/too-many-parens/run.t

```diff
@@ -19,3 +24,8 @@ are readable.
          ^^^^^^^
   These parentheses are no longer necessary with dune, please remove them.
   [1]
+
```

Comment:
Can you add a short message saying that you're testing that extra long files are not printed?

Reply:
done